docs/topics/rn-new-features.adoc

```diff
@@ -7,66 +7,22 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.Updated {WebName} interface
+.MTA plugin for IntelliJ IDEA
 
-The current release contains an updated {WebName} interface with improved project management features and additional transformation targets.
+The current release supports the {ProductName} plugin for the IntelliJ IDEA IDE. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort. It has the same functionality as other {ProductName} plugins: it analyzes your projects using customizable rulesets, marks migration issues in the source code, and provides guidance to fix issues.
```

Comment:
"the IntelliJ IDEA IDE" > "IntelliJ IDEA"

"It has the same functionality as other {ProductName} plugins:" - Suggest you remove this. Fluff.